modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -73,6 +73,9 @@ Anypoint Runtime Fabric requires one of the following operating systems:
 
 * Red Hat (RHEL) v7.4, v7.5, v7.6, v7.7, v7.8
 * CentOS v7.4, v7.5, v7.6, v7.7
++
+[IMPORTANT]
```

Comment:
I would suggest to change the wording to following:
[Note]
The Runtime Fabric install-script include AMIs with the default kernel version which requires kernel to be upgraded to version 3.10.0-1075 or above for the product stability requirement.

FYI @sundippatel if you have better suggestion to include this message to inform customer to upgrade kernel version for the memory issues in older kernels.

Reply:
The update is intended to remind customers to use new kernels even with their own infrastructure. 

The below seems more appropriate on the script download page
`[Note]
The Runtime Fabric install-script include AMIs with the default kernel version which requires kernel to be upgraded to version 3.10.0-1075 or above for the product stability requirement.`